RefrigeratorExe/RefrigeratorExe/Refrigerator.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace RefrigeratorExe
+{
+    internal class Refrigerator :IComparable<Refrigerator>
+    {
+        private string _model;
+
+
+        public Guid id { get; }
```

Comment:
Id
Naming conventions: Prefer capital letter if this is a public property.

Good thing: the use of Guid